templates/public-cloud/index.html

```diff
@@ -44,36 +44,36 @@ <h2 class="p-heading--three">Fast service deployment with&nbsp;Juju</h2>
       <p><a class="p-link--external" href="https://jujucharms.com/">Learn more about Juju</a></p>
     </div>
     <div class="col-6 p-divider__block">
-      <h2 class="p-heading--three">The best management and&nbsp;support</h2>
+      <h2 class="p-heading--three">Trusted Container (OCI) Images</h2>
       <div class="u-hide--small u-align--center u-sv1">
         {{
           image(
-            url="https://assets.ubuntu.com/v1/434e177d-cloud-landscape-chart.jpg",
-            alt="Landscape managing 913 machines screenshot",
+            url="https://assets.ubuntu.com/v1/fcb97fce-canonical-on-amazon-ecr.png",
+            alt="Canonical on Amazon screenshot",
             height="208",
             width="371",
             hi_def=True,
             loading="lazy",
           ) | safe
         }}
       </div>
-      <p>The Ubuntu Advantage support service has flexible Ubuntu Virtual Guest options, designed for virtualised enterprise workloads on Ubuntu-certified public clouds. Thanks to Landscape, efficient management of all your Ubuntu instances is only a few clicks away. And with the Canonical Livepatch Service, you can apply critical kernel patches without rebooting.</p>
-      <p><a href="/support">Find out all about support&nbsp;&rsaquo;</a></p>
+      <p>The provenance of base and application images is critical for security. While tooling like Docker makes retrieving and running images trivial, ensuring a trusted software supply chain is required. A simple way to get started is by using our hardened repositories. We maintain a set of stable & secure (LTS) tracks for application, runtime, and base container images. Get the best of the Ubuntu developer experience and commercial commitments for enterprise containers.</p>
+      <p><a href="/support">Explore Canonical's LTS Docker Images Portfolio&nbsp;&rsaquo;</a></p>
     </div>
   </div>
 </section>
 
 <section class="p-strip--light is-bordered">
   <div class="row u-equal-height u-vertically-center">
     <div class="col-5">
-      <h2>Developers' favourite</h2>
-      <p>Ubuntu is the most popular Linux distribution for development and deployment. If your developers already use Ubuntu, moving to the cloud is even easier. </p>
-      <p><a href="/desktop/developers">Learn more about Ubuntu for developers&nbsp;&rsaquo;</a></p>
+      <h2>Managed Applications</h2>
+      <p>Accelerate innovation across clouds with fully managed Apache Kafka, Kubeflow, ElasticSearch and more. We build and operate the solution for you, backed by a full SLA. With Canonical’s managed open source application services there’s no lock in and you can take over control whenever you want.</p>
```

Comment:
```suggestion
      <p>Accelerate innovation across clouds with fully managed Apache Kafka, Kubeflow, ElasticSearch and more. We build and operate the solution for you, backed by a full SLA. With Canonical's managed open source application services there's no lock in and you can take over control whenever you want.</p>
```
Sometimes in the copydoc the apostrophes (and sometime quotation marks) used will be angled but we use the straight ones in our code :)